source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs

```diff
@@ -776,9 +776,8 @@ public void BadImageWidthsAsAdditionalProperties()
 
             var result = AdaptiveCard.FromJson(payload);
 
-            // Expect one warning for each unknown property (3)
             // and one for each bad pixel size (2)
```

Comment:
>and one for each bad pixel size (2) [](start = 15, length = 35)

[nit] Could you update this comment to be a complete sentence? ("Expect one warning for each...")